.github/workflows/build.yaml

```
@@ -77,6 +77,14 @@ jobs:
 
           cp ./scripts/livepeer-vmagent ./bin/livepeer-vmagent
 
+      - name: Build c2patool
+        run: |
+          apt update && apt install -y curl build-essential git
+          curl https://sh.rustup.rs -sSf | sh -s -- -y
+          source "$HOME/.cargo/env"
+          cargo install c2patool
```

Probably this to match the other one?
```suggestion
          cargo install --version 0.6.2 c2patool
```

---

https://github.com/marketplace/actions/cargo-install I find dedicated github actions working faster than manual installations. Maybe worth trying out